src/operator/nn/fully_connected.cc

```diff
@@ -210,17 +210,17 @@ inline static bool BackwardFCStorageType(const nnvm::NodeAttrs& attrs,
   CHECK_EQ(in_attrs->size(), 3U);
   CHECK_EQ(out_attrs->size(), out_expected);
 
-  DispatchMode wanted_mode;
-#if 0
+  bool dispatched = false;
   // TODO(zhengda) let's disable MKLDNN for FullyConnected for now.
```

Comment:
@zheng-da @azai91  is there a plan to enable FC with MKLDNN, or at least test it out? If there's a bug reveal by this test, this will also help stabilize mxnet-MKLDNN.

Reply:
ideally we should enable it. i forgot the case that it fails. i don't think MKLDNN FC backward is much faster than our native implementation. so it's not very urgent.
@pengzhao-intel @TaoLv did you check if MKLDNN FC backward works in all cases?

Reply:
We have enabled this in the #11301 and it passed all unit tests and coverage test in our local.
So, I think MKLDNN FC backward works fine.